lib/defra_ruby/area/response.rb

```diff
@@ -4,11 +4,11 @@ module DefraRuby
   module Area
     class Response
       attr_reader :error
-      attr_reader :area
+      attr_reader :areas
 
       def initialize(response_exe)
         @success = true
-        @area = nil
+        @areas = nil
```

Comment:
I do believe that since we moved from `string` to `array` we should give the user back an empty array rather than a `nil` value. This will allow the user to trust using array methods to check the presence without having to deal with `nil` which does not respond to methods suc `size`, `lenght`.... :)

Reply:
Awesome suggestion. Will do!